<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <widget class="QWidget" name="program">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="search_product">
             <attribute name="title">
              <string>Search</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QGridLayout" name="search_grid">
                <item row="0" column="0">
                 <widget class="QWidget" name="search_widget" native="true"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="new_product">
             <attribute name="title">
              <string>New</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QGridLayout" name="new_grid">
                <item row="0" column="0">
                 <widget class="QWidget" name="new_widget" native="true"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="login">
             <attribute name="title">
              <string>Login</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QWidget" name="credential_holder" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="loading">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_loading">
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>600</height>
                </size>
               </property>
               <property name="text">
                <string>Loading...</string>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
